kotsadm/web/src/components/SecureAdminConsole.jsx

```diff
@@ -235,6 +249,16 @@ class SecureAdminConsole extends React.Component {
     } = this.state;
 
     if (fetchingMetadata || !loginInfo) {
+      if (loginErr && loginErrMessage) {
```

Comment:
would prefer it if this was outside in its own block, before this condition `if (fetchingMetadata || !loginInfo)`